modules/sdk/src/main/scala/org/tessellation/sdk/domain/healthcheck/consensus/HealthCheckConsensusRound.scala

```diff
@@ -70,6 +70,9 @@ class HealthCheckConsensusRound[F[_]: Async, K <: HealthCheckKey, A <: HealthChe
     }
   }
 
+  def hasProposal(peer: PeerId): F[Boolean] =
```

Comment:
I think it should be more refined and it should tell wether that is the same proposal, or it's a different proposal but for the same `RoundId`s or it's a different proposal without relation to `RoundId`s of this healthcheck. Based on this information the proposal should be handled differently.

Reply:
IMO info whether round has a proposal from peer or no is enough. Proposal can be handled within the round by itself and from "manager" point of view I'd just need to know whether pass proposal to the round itself or no.

Reply:
If you ask the round for more details you may find the abnormal situation, like sending two different proposals for the same round from the same peer - this could also be checked for the historical round instead (the moment it becomes historical), so I guess it's ok. But in general I would rather know and figure out if we have an issue then to just let such proposals fly by.